app/Console/Commands/indexEntities.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Movie;
+use Illuminate\Console\Command;
+
+class indexEntities extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'elastic:index';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Command description';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        
+        $mappingProperties = array(
+            'title' => array(
+                 'type' => 'keyword'
+             )
+        );
+        Movie::createIndex();
+        Movie::putMapping($mappingProperties);
+        Movie::chunk(Movie::count(), function ($movies) {
```

Comment:
`Movie::count()` ce ti vratiti broj muvija u bazi, sto znaci da ces ih sve dovuci iz jednog puta
tu treba da stoji tipa 100 ili 1000 ili sta god ali tako neki broj (u realnim uslovima to bi se testiralo sta je najbolje staviti)